packages/components/src/animation/features/index.ts

```diff
@@ -0,0 +1,15 @@
+const loadFramerFeatures = async ( feature: string ) => {
```

Comment:
[nit] I'd prefer to separate them into two functions but I'm okay with the current way.

For example,
```js
export const loadDomAnimation = () => import( './domAnimation' ).then( mod => mod.default );

export const loadDomMax = () => import( './domMax' ).then( mod => mod.default );
```